nistats/first_level_model.py

```diff
@@ -515,12 +515,12 @@ def compute_contrast(self, contrast_def, stat_type=None,
 
         output_type : str, optional
             Type of the output map. Can be 'z_score', 'stat', 'p_value',
-            'effect_size' or 'effect_variance'
+            'effect_size', 'effect_variance' or 'all'
 
         Returns
         -------
-        output : Nifti1Image
-            The desired output image
+        output : Nifti1Image or dict
```

Comment:
This is the only thing I dislike in this PR: that the output type depends on a rather obscure way on the input.
Yet I cannot think of any way to avoid this.
I think I would at least state here that the output type is conditional on `output_type` being `all`